src/cascadia/TerminalApp/App.cpp

```diff
@@ -159,13 +162,76 @@ namespace winrt::TerminalApp::implementation
         {
             IInspectable g = res.Lookup(key);
             winrt::Windows::UI::Xaml::Style style = g.try_as<winrt::Windows::UI::Xaml::Style>();
+            _root.Style(style);
             _tabRow.Style(style);
         }
 
         // Apply the UI theme from our settings to our UI elements
         _ApplyTheme(_settings->GlobalSettings().GetRequestedTheme());
 
         _OpenNewTab(std::nullopt);
+
+        _root.Loaded({ this, &App::_OnLoaded });
+    }
+
+    // Method Description:
+    // - Show a ContentDialog with a single "Ok" button to dismiss. Looks up the
+    //   the title and text from our Resources using the provided keys.
+    // - Only one dialog can be visible at a time. If another dialog is visible
+    //   when this is called, nothing happens.
+    // Arguments:
+    // - titleKey: The key to use to lookup the title text from our resources.
+    // - textKey: The key to use to lookup the content text from our resources.

```

Comment:
Out of curiosity, did you consider naming this 'contentKey' instead of 'textKey' given the description of the parameter? Not asking for a change, just curious since I'm trying to understand naming patterns in the project.

Reply:
Oh, yea I guess that `contentKey` makes a little more sense. I think this was just an oversight while I was writing it. I can change it.

Reply:
Did you accidentally reverse this change?  titleKey is now contentKey and textKey is still textKey?

Reply:
Yes, yes I did. That's my bad.